docs/literate/src/files/adding_new_scalar_equations.jl

```diff
@@ -101,8 +102,9 @@ plot!(sol)
 # Now let's increase the final time (and also the spatial resolution).
 
 ## A larger final time: Nonclassical shocks develop (you can even increase the refinement to 12)
-semi = remake(semi, mesh=TreeMesh(-1.0, 1.0, initial_refinement_level=8, n_cells_max=10^5))
-ode = semidiscretize(semi, (0.0, 0.5) #= tspan =#)
+semi = remake(semi,
+              mesh = TreeMesh(-1.0, 1.0, initial_refinement_level = 8, n_cells_max = 10^5))
+ode = semidiscretize(semi, (0.0, 0.5)) #= tspan =#
```

Comment:
```suggestion
ode = semidiscretize(semi, (0.0, 0.5))
```
Same here.

Reply:
I think it would be nice to keep a comment here above the line as well